dgraph/cmd/zero/raft.go

```diff
@@ -80,8 +84,19 @@ func (n *node) AmLeader() bool {
 	return time.Since(n.lastQuorum) <= 5*time.Second
 }
 
+// {2 bytes Node ID} {4 bytes for random} {2 bytes zero}
```

Comment:
NIT: this is duplicated in the worker package (below).  The proposalKey concept could be implemented in the x package or elsewhere... not a deal-breaker...